tests/linux_kernel/helpers/test_runq.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025, Oracle and/or its affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+import time
+
+from drgn.helpers.linux.pid import find_task
+from drgn.helpers.linux.runq import task_lastrun2now
+from tests.linux_kernel import LinuxKernelTestCase, fork_and_stop
+
+
+class TestRunq(LinuxKernelTestCase):
+    def sleep_10ms():
+        time.sleep(0.01)
+
+    def test_task_lastrun2now(self):
+        with fork_and_stop(self.sleep_10ms) as pid:
```

Comment:
If the forking process is the one sleeping, then it will sleep for 0.01 seconds, then wake up and run again, then stop, so I don't think this tests what you want. I think you want the main process to do the sleep so that the forked process stays stopped for at least that long:

```suggestion
        with fork_and_stop() as pid:
            time.sleep(0.01)
```

Reply:
Yes, this makes more sense.